### PR TITLE
Update imagej to 1.53

### PR DESCRIPTION
**Pull request by takameron**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

### CODE REVIEW

File: Casks/imagej.rb [hunk 1]
```diff
@@ -1,15 +1,29 @@
...
+
+  if Hardware::CPU.intel?
+    sha256 "4093259645569f5dc04ef5576a5f54ab878c0923f48c03b866f5d56f2edc078f"
+
+    url "https://wsr.imagej.net/distros/osx/ij#{version.no_dots}-osx-java8.zip",
+        verified: "wsr.imagej.net/distros/osx/"
+  else
+    sha256 "e464760174af0bcff7849db6e12090a4cbb1b0a1897a71dc1a999855d6e22e98"
+
+    url "https://wsr.imagej.net/distros/osx/ij#{version.no_dots}-osx-arm-java13.zip",
```
Comment (gdams) [inline Casks/imagej.rb:12]: Is there a reason that Zulu 13 was chosen to be bundled here? Java 13 is unsupported upstream anymore?

Comment (takameron) [reply Casks/imagej.rb:12]: [The download page](https://imagej.nih.gov/ij/download.html) only listed Java 13 bundles. However, Java 8 bundles are also [distributed](http://wsr.imagej.net/distros/osx/). Should I choose the one that bundles Java 8?

Comment (bevanjkay) [reply Casks/imagej.rb:12]: @takameron We should distribute the version provided by upstream, which in this case look like the preference is the java13 variant as seen on [the download page](https://imagej.nih.gov/ij/download.html).

I would like to have a quick look at the livecheck block before merging.

**Comment by bevanjkay** [comment]
Thanks @takameron

**Pull request merged**